qa/tasks/cephfs/kernel_mount.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, ctx, test_dir, client_id, client_remote,
             client_keyring_path=client_keyring_path, hostfs_mntpt=hostfs_mntpt,
             cephfs_name=cephfs_name, cephfs_mntpt=cephfs_mntpt, brxnet=brxnet)
 
-        self.rbytes = config.get('rbytes', False)
+        self.rbytes = None if config is None else config.get('rbytes', False)
```

Comment:
How about pass the `rbytes` instead here ? Since the `config` parameter only used for getting the `rbytes`.

Reply:
What do you mean "here"? `KernelMount.__init__`? If so, umm, I think having `config` is better from the perspective that in future if someone else requires another value from `config`, the person would have to just `get()` that value from the config instead of having to modify all KernelMount() calls to pass another value.

Reply:
Yeah, in the `KernelMount.__init__(..., config=None)`.  Not sure, maybe that won't happen. I am okay with this change too, it will up to you :-)

Reply:
> Or possibly we could set it like config={} in __init__() ?

+1

Reply:
@lxbsz Yes, that's much better. Update the branch.